src/main/java/com/yurjinia/auth/service/AuthService.java

```diff
@@ -43,10 +43,35 @@ public class AuthService {
     public JwtAuthenticationResponse signUp(RegistrationRequest registrationRequest, MultipartFile image) {
         userService.createUser(registrationRequest, image);
 
+        String token = confirmationTokenService.createToken(registrationRequest.getEmail());
+
+        String confirmationLink = "http://localhost:9000/api/v1/auth/sign-up/confirm?token=" + token;
+
+        sendConfirmationEmail(registrationRequest.getEmail(), confirmationLink);
+
         final var jwt = jwtService.generateToken(registrationRequest.getEmail());
         return new JwtAuthenticationResponse(jwt);
     }
 
+    private void sendConfirmationEmail(String email, String confirmationLink) {
+        String emailBody = "Для підтвердження вашої реєстрації перейдіть за посиланням: " + confirmationLink;
```

Comment:
Ukrainian to English.

Reply:
check the email structure, how it was done in other features.
it must be covered in HTML, similarly as other template messages.